src/ucp/wireup/wireup.c

```diff
@@ -190,8 +191,15 @@ static void ucp_wireup_process_request(ucp_worker_h worker, const ucp_wireup_msg
         }
     }
 
+    /* Use default parameters to initialize local lanes.
+     * This might not always work - need to pass some parameters in the wireup
```

Comment:
I'm not sure that increase of error handling mode greater than requested is correct. I think we need to reconfigure existing EP if it was created with default config/key on wire up but requested something else.